lib/tty/logger.rb

```diff
@@ -204,6 +204,16 @@ def log?(level, other_level)
       compare_levels(level, other_level) != :gt
     end
 
+    # Log streaming output. Assumes :debug level. 
+    #
+    # @example
+    #   logger << "Example output"
+    #
+    # @api public
+    def <<(*msg)
+      log(:debug, *msg)
```

Comment:
Would it be possible to use the configured level here? I believe this to be more intuitive and respecting user's preference. Anything stopping us from doing this?

Reply:
IIRC, the configured level is the minimum level of events which `tty-logger` will output to console, right? I used `:debug` because otherwise the event level would be unpredictable.

Reply:
I may be completely wrong here so please bear with me. Doing many things at the same time. 

As I see it, if this method is changed to, instead of the `:debug` for all level, to the level that a logger is configured with, the `<<` call will respect it and write at that requested level - using the correct label. I could see a problem only if we allowed setting the level via the `<<` method. In other words, to me, the behaviour is predictable as the requested level is applied to all messages sent from the caller. Otherwise, the user would be really perplexed when they requested only the `:fatal` level to be logged and everything is printed with `:debug` level labels.

Another thing I would probably consider is changing the method name to `write` and making the `<<` an alias to mimic Ruby's IO standard library.